changelog/unreleased/improve-ocmd-error-logs.md

```diff
@@ -0,0 +1,6 @@
+Enhancement: Improve error logging in ocmd flow
+
+https://github.com/cs3org/reva/pull/3524
```

Comment:
To pass the changeling CI, just update this line to https://github.com/cs3org/reva/pull/3524